lib/travis/api/v3/queries/installation.rb

```diff
@@ -0,0 +1,10 @@
+module Travis::API::V3
+  class Queries::Installation < Query
+  	params :id
+
+  	def find
+      return Models::Installation.where(github_id: id) if id
```

Comment:
`where` will return a collection, this should be `Models::Installation.find_by_gthub_id(github_id)` or `Models::Installation.find_by(gthub_id: github_id)`